pkgs/development/python-modules/invoice2data/default.nix

```diff
@@ -9,15 +9,17 @@ buildPythonPackage rec {
     sha256 = "0p3ka85n0qmdkrjffvqgcj3pbl002srrlcjdk8pyhhwg6b78d4l7";
   };
 
+  makeWrapperArgs = ["--prefix" "PATH" ":" "${tesseract}/bin/tesseract:${imagemagick}/bin/convert:${xpdf}/bin/pdftotext" ];
+
   propagatedBuildInputs = [ unidecode dateparser pyyaml pillow chardet ];
 
   # Tests fails even when ran manually on my ubuntu machine !!
   doCheck = false;
 
   meta = with stdenv.lib; {
-    description = "an e-mail encryption, virus- and spam- checking module, gateway and daemon";
-    homepage = https://github.com/gpgmailencrypt/gpgmailencrypt;
-    license = licenses.gpl3;
+    description = "Data extractor for PDF invoices - invoice2data";
+    homepage = https://github.com/invoice-x/invoice2data;
+    license = licenses.mit;
```

Comment:
I missed the meta block in my last review. Can you squash that into the first commit? (or squash everything together)